test/integration/targets/vyos_firewall_interfaces/tests/cli/merged.yaml

```diff
@@ -0,0 +1,75 @@
+---
+- debug:
+      msg: "START vyos_firewall_interfaces merged integration tests on connection={{ ansible_connection }}"
+
+- include_tasks: _populate_rule_sets.yaml
+
+- include_tasks: _remove_config.yaml
+
+- block:
```

Comment:
Please add a test for updating existing attributes using `state: merged`.

Reply:
Done